crypto/test/src/security/test/TestParameterUtil.cs

```diff
@@ -67,7 +67,7 @@ public void TestCreateKeyParameter()
 			Type			expectedType,
 			byte[]			expectedBytes)
 		{
-			Assert.IsTrue(expectedType.IsInstanceOfType(key));
+			Assert.IsTrue(expectedType.GetTypeInfo().IsInstanceOfType(key));
```

Comment:
Can't this just be expectedType.IsIstanceOfType(key)

Reply:
Depending on the version of NETStandard, some of these methods are on a `TypeInfo` rather than the `Type`. It's something I had to deal with when porting my own libraries to .NETStandard.

Reply:
This could probably be simplified now that I've removed < NET Standard 2.0. NS 2 should have the full Type surface area.

Reply:
I was hoping you'd keep .NETStandard < 2.0 because MimeKit targets 1.3, 1.6, and 2.0

Reply:
I can't since the original codebase used Array.Copy that takes a long and that overload doesn't exist before netstandard 2.

I'd strongly suggest nuking 1.3 and 1.6 as literally nothing Microsoft supports targets below 2.0 anymore. 

The lowest supported UWP is 16299 and that's netstandard 2.0. .NET Core 1.0, 1.1, and 2.0 are all out of support. There's .NET 4 multi-target, so all current .net framework's are covered.

Reply:
Okay, thanks for the info.